src/test/objectstore/store_test.cc

```diff
@@ -9212,6 +9213,82 @@ TEST_P(StoreTestSpecificAUSize, BluestoreBrokenZombieRepairTest) {
   bstore->mount();
 }
 
+TEST_P(StoreTestSpecificAUSize, BluestoreRepairSharedBlobTest) {
+  if (string(GetParam()) != "bluestore")
+    return;
+  if (smr) {
+    cout << "TODO: repair mismatched write pointer (+ dead bytes mismatch)" << std::endl;
```

Comment:
"TODO: untested for smr" ?

Reply:
it's a copy paste from another test case which Sage wrote. ;) Hence leaving as-is.